File: fizzbuzz/Fortran/fizzbuzz.f90
! Compile with gfortran fizzbuzz.f90 and run with ./a.out
! Standard fizzbuzz problem from 1 to 100
! Program incompatible with prior versions of fortran due to do-loop structure


program FizzBuzz

implicit none

integer n

do n = 1, 100
    if (mod(n, 3) == 0 .and. mod(n, 5) == 0) then
        write (*,'(a)') 'FizzBuzz'
    else if (mod(n, 3) == 0) then
        write (*,'(a)') 'Fizz'
    else if (mod(n, 5) == 0) then
        write (*,'(a)') 'Buzz'
    else
        write (*,'(i0)') n
    end if
end do

end program FizzBuzz

File: fizzbuzz/VB.NET/fizzbuzz.vb
Imports System

Module Program
    Sub Main(args As String())
        For i = 1 To 100
            If i Mod 3 = 0 And i Mod 5 = 0 Then
                Console.WriteLine("FizzBuzz")
            ElseIf i Mod 3 = 0 Then
                Console.WriteLine("Fizz")
            ElseIf i Mod 5 = 0 Then
                Console.WriteLine("Buzz")
            Else
                Console.WriteLine(i)
            End If
        Next
    End Sub
End Module
